### ISSUE TITLE
Bug: Error when folder is selected in `FzfxBuffers`

### ISSUE THREAD

**Issue by savchenko** [linrongbin16/fzfx.nvim#738]
### Steps to reproduce

1. Have NetRW open so that one of the buffers is a folder
2. Call `:FzfxBuffers` and select the folder
3. Observe the error below

```lua
E784: Cannot close last tab page                                                                                                                                                                                     
Error executing luv callback:                                                                                                                                                                                        
...al/share/nvim/lazy/fzfx.nvim/lua/fzfx/commons/fileio.lua:251: failed to read file "toolbox": "EISDIR: illegal operation on a directory"                                                                           
stack traceback:                                                                                                                                                                                                     
        [C]: in function 'error'                                                                                                                                                                                     
        ...al/share/nvim/lazy/fzfx.nvim/lua/fzfx/commons/fileio.lua:251: in function <...al/share/nvim/lazy/fzfx.nvim/lua/fzfx/commons/fileio.lua:249>
```

Potentially related to https://github.com/linrongbin16/fzfx.nvim/issues/734

**Comment by linrongbin16** [comment]
hi @savchenko , I merged PR #739 to fix this issue.

But since I actually don't have a local environment to test it with `netrw`, so I'm not 100% confident it's been fixed. Please pull and try.

**Comment by linrongbin16** [comment]
> It actually doesn't list NetRW folders at all now. Perfect!

I don't think I changed the providers, i.e. the NetRW list should remain the same results 🤣 .

Anyway, let's keep it now.
